consultation_analyser/hosting_environment.py

```diff
@@ -18,6 +18,11 @@ def is_deployed() -> bool:
         deployed_envs = ["DEV", "DEVELOPMENT", "PREPROD", "PROD", "PRODUCTION"]
         return environment in deployed_envs
 
+    @staticmethod
+    def is_production() -> bool:
+        environment = env.str("ENVIRONMENT", "").upper()
+        return environment in ["PROD", "PRODUCTION"]
```

Comment:
Why do we have 2? Can't we have just `PRODUCTION`?

Reply:
Just checked in the infra repo - it's "Prod" so I'll update.